lib/smart_contracts/qx_info.dart

```diff
@@ -10,4 +10,6 @@
   static const issueAssetInputType = 1;
   /// Input type for transferring an asset
   static const transferAssetInputType = 2;
+  static const qutilAddress = "EAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVWRF";
```

Comment:
@ahmed-tarek-salem 
Qx is one smart contract.
QUtil is another smart contract.

Reply:
Resolved.